src/eko/output/struct.py

```diff
@@ -10,7 +11,7 @@
 import tarfile
 import tempfile
 import time
-from dataclasses import dataclass, field, fields
+from dataclasses import dataclass
```

Comment:
since you went for `dataclasses` globally, let's drop also this one

Reply:
On this I'm plainly inconsistent :D